frontend/apps/reader/readerui.lua

```diff
@@ -342,6 +349,16 @@ function ReaderUI:init()
     end
     self.postInitCallback = nil
 
+    -- Now that document is loaded, store book metadata in settings
+    -- (so that filemanager can use it from sideCar file to display
+    -- Book information).
+    -- via pcall because picdocument:getProps() may fail
+    local ok, doc_props = pcall(self.document.getProps, self.document)
+    if not ok then
+        doc_props = {}
+    end
+    self.doc_settings:saveSetting("doc_props", doc_props)
```

Comment:
I would :| It would need us to manage another read/save/flush setting.
It's already there in the current settings, under the ``stats`` key. I'm just easing the transition when statistics plugin won't update this stats key anymore. There's one single file today, I don't want to be the one introducing a second file :) (also see note above about `DocSettings:saveSetting()` only updating a key, and not flushing to disk).
(My aim is to add features, while keeping code clean and neat, as it is already. Not so much about a big refactoring. I'm quite often just cut and pasting from other existing code that works :)